cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageImpl.kt

```diff
@@ -99,8 +99,15 @@ private class DataStorageImpl(
      * By default - false (if there are no ccpaConsentStatus in data storage or applies value is null)
      */
     override val ccpaApplies: Boolean
-        get() = ccpaConsentStatus?.let { verifiedCcpaConsentStatusString ->
-            check { JsonConverter.converter.decodeFromString<CcpaCS>(verifiedCcpaConsentStatusString) }
+        get() = metaDataResp?.let { metaData ->
```

Comment:
I like that, since the most relevant info about the campaigns applies value is indeed in the `/meta-data` response, it makes perfect sense to depend on that